Gemfile

```diff
@@ -25,6 +25,8 @@ end
 
 gem 'plek', '1.7.0'
 
+gem 'govuk-content-schema-test-helpers', '~> 1.1.0'
```

Comment:
I can see you configure it in test/test_helper.rb, but I can't see where you're actually using the gem.